ipatests/test_integration/test_server_del.py

```diff
@@ -71,19 +71,31 @@ def check_removal_disconnects_topology(
 
 class ServerDelBase(IntegrationTest):
     num_replicas = 2
-    num_clients = 1
+    num_clients = 0
     domain_level = DOMAIN_LEVEL_1
     topology = 'star'
 
     @classmethod
     def install(cls, mh):
         super(ServerDelBase, cls).install(mh)
-
-        cls.client = cls.clients[0]
         cls.replica1 = cls.replicas[0]
         cls.replica2 = cls.replicas[1]
 
 
+def remove_segment(master, left, right, suffix=DOMAIN_SUFFIX_NAME):
+    """
+    broke topology as in described in install method
```

Comment:
A more precise description would be:
"""
Ensure that segment left-right or right-left don't exist
"""

As the method doesn't do anything based on the described topology.